sky/tools/sky_snapshot/vm.cc

```diff
@@ -26,10 +28,15 @@ static const char* kDartArgs[] = {
 
 void InitDartVM() {
   CHECK(Dart_SetVMFlags(arraysize(kDartArgs), kDartArgs));
-  CHECK(Dart_Initialize(
-            reinterpret_cast<uint8_t*>(&kDartVmIsolateSnapshotBuffer), nullptr,
-            nullptr, nullptr, nullptr, nullptr, nullptr, nullptr, nullptr,
-            nullptr, nullptr, nullptr, nullptr, nullptr) == nullptr);
+  char* init_message = Dart_Initialize(
+      reinterpret_cast<uint8_t*>(&kDartVmIsolateSnapshotBuffer), nullptr,
+      nullptr, nullptr, nullptr, nullptr, nullptr, nullptr, nullptr, nullptr,
+      nullptr, nullptr, nullptr, nullptr);
+  if (init_message != nullptr) {
+    std::cerr << "Dart_Initialize Error: " << init_message << std::endl;
```

Comment:
Dang, I was trying to be consistent with https://github.com/flutter/engine/blob/master/sky/tools/sky_snapshot/main.cc#L22